lib/inspec/profile.rb

```diff
@@ -263,6 +272,7 @@ def info(res = params.dup) # rubocop:disable Metrics/CyclomaticComplexity
 
       res
     end
+    # rubocop:enable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
I don't think we need this line, when putting it after a method call it is restricted to only that method.

Reply:
Turns out you can't do it on a method, and its file based.